### ISSUE TITLE
Blank page with IPAD (and not with IPAD PRO)

### ISSUE THREAD

**Issue by Pitoucol** [projectestac/moodle-mod_geogebra#94]
We have problem with 3D rendering objects in safari for IPAD. Only the classic version, not the pro version. 
When loading activity with 3D rendering (sphere for exemple), the rendering frame is blank. (we have control button, to reload or change)
One of the test Ipad is on IOS 17.0.2. Problem occurs with safari, chrome or firefox.
![image](https://github.com/projectestac/moodle-mod_geogebra/assets/70409859/b25ee8f7-558c-4f2b-a5c7-eb2f856e80be)

On IPAD pro, rendering is ok (IOS 17.0.3).

![image](https://github.com/projectestac/moodle-mod_geogebra/assets/70409859/afbe89c3-7c4a-415d-ab4a-231cdd8930aa)

**Comment by murkle** [comment]
Do you have a link to the .ggb file?

**Comment by Pitoucol** [comment]
Sorry, I forgot this essential point : 
Here is the link on geogebra website https://www.geogebra.org/m/fBznctJg
Here
[material-b2cwWvm9.zip](https://github.com/projectestac/moodle-mod_geogebra/files/13452336/material-b2cwWvm9.zip)
is the ggb file we use in moodle

**Comment by murkle** [comment]
Thanks, what's the iOS version on the iPad Pro where it does work?

**Comment by Pitoucol** [comment]
Hello. 
Ipad PRO is IOS 17.0.3, but the problem appeared with older versions, last year in April-May.
So it's not specifically related to one particular version. 

Can you reproduce this problem by your side ?

**Comment by murkle** [comment]
Yes, https://www.geogebra.org/m/qxtcvbkj doesn't load on my iPad6

Please send any other problem .ggb files. Thanks 🙏

**Comment by Pitoucol** [comment]
Here you are  ;-)
[https://www.geogebra.org/m/ZqBpSATW](url)
[https://www.geogebra.org/m/jbgudrsz](url)

**Comment by murkle** [comment]
Which model iPad has the problem?

**Comment by Pitoucol** [comment]
For the moment, Ipad 7. But I will look forward on monday to check with others ipad from my school.

**Comment by Pitoucol** [comment]
3D rendering view works well on IPAD 9 and Iphone 14 and 14 PRO

**Comment by murkle** [comment]
Thanks again! We'll have a look but it's going to be a tricky one to track down...

**Comment by murkle** [comment]
@Pitoucol Apple seem to have fixed the bug in iOS 17.2 - please can you test that

**Comment by Pitoucol** [comment]
Hello
Good news. tested yesterday with an Ipad 7, upgraded to 17.2, and yes, problem solved.
